docs/source/examples/demo_tracing_model_torchscript_onnx.ipynb

```diff
@@ -0,0 +1,463 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "fba4c02d",
+   "metadata": {},
+   "source": [
+    "# Demo Notebook to trace Sentence Transformers model\n",
+    "\n",
+    "This notebook provides a walkthrough guidance for users to trace models from Sentence Transformers in torchScript and onnx format. After tracing the model customer can upload the model to opensearch and generate embeddings.\n",
+    "\n",
+    "Step 0: Import packages and set up client\n",
+    "\n",
+    "Step 1: Save model in torchScript format\n",
+    "\n",
+    "Step 2: Upload the saved torchScript model in Opensearch\n",
+    "\n",
+    "Step 3: Save model in Onnx format\n",
+    "\n",
+    "Step 4: Upload the saved Onnx model in Opensearch\n",
+    "\n",
+    "Step 5: Generate Sentence Embedding with uploaded models\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7011727e",
+   "metadata": {},
+   "source": [
+    "## Step 0: Import packages and set up client\n",
+    "Install required packages for opensearch_py_ml.sentence_transformer_model\n",
+    "Install `opensearchpy` and `opensearch-py-ml` through pypi\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "17a3e085",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "#!pip install opensearch-py opensearch-py-ml"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "d0c711bf",
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "warnings.filterwarnings('ignore')\n",
```

Comment:
Is this must have? What warning we will ignore?

Reply:
This is not must have. We had this `filterwarnings` from the beginning. This mostly ignore all the warnings including like `DeprecationWarning`. I'll look into this to relax this warning if I need to then.